topiary/src/configuration.rs

```diff
@@ -41,20 +39,32 @@ impl Configuration {
                         ));
                     }
                 }
-                "indent-level" => {
-                    if let Some(arg) = arguments.next() {
-                        indent_level = arg.parse::<usize>().map_err(|_| {
+                "indent" => {
+                    if let Some(arg) = argument {
+                        // Strip first and last " or ' from the string
+                        // and unescape characters like \t
+                        let arg = unescape(&arg[1..arg.len() - 1]).ok_or_else(|| {
                             FormatterError::Query(
                                 format!(
-                                    "The #indent-level! parameter must be a positive integer, but got '{arg}'"
+                                    "The #indent! parameter could not be unescaped, got '{arg}'"
                                 ),
                                 None,
                             )
                         })?;
+
+                        if arg.chars().all(char::is_whitespace) {
+                            indent = arg;
+                        } else {
+                            return Err(FormatterError::Query(
+                                format!(
+                                    "The #indent! parameter must only contain whitespace, but got '{arg}'"
+                                ),
+                                None,
+                            ));
```

Comment:
Aw, no emoji indentation :disappointed:

Reply:
Want to remove the check? I'd be happy to, it seems arbitrarily limiting anyway.

Reply:
No, I'm just joking :slightly_smiling_face: Limiting to whitespace is fine.